BootloaderCommonPkg/Include/Library/BootloaderCommonLib.h

```diff
@@ -131,7 +133,8 @@ GetComponentHash (
 RETURN_STATUS
 SetComponentHash (
   IN       UINT8             ComponentType,
-  IN CONST UINT8            *HashData
+  IN CONST UINT8            *HashData,
+  IN       UINT8             HashType
```

Comment:
How about defining a enum variable for HashType instead of using UINT8?

Reply:
Yeah, i would add enum variables as suggested.
Also, for other supported ones in cryptolib.

Reply:
Guo, I would add the change hash types, signing types and other definitions in Crypto as a separate patch.